nixos/modules/system/boot/plymouth.nix

```diff
@@ -143,17 +152,22 @@ in
       sed -i '/loginctl/d' $out/71-seat.rules
     '';
 
-    # We use `mkAfter` to ensure that LUKS password prompt would be shown earlier than the splash screen.
-    boot.initrd.preLVMCommands = mkAfter ''
-      mkdir -p /etc/plymouth
+    boot.initrd.preLogCommands = ''
+      mkdir -p /etc/plymouth /usr/share
       ln -s ${configFile} /etc/plymouth/plymouthd.conf
       ln -s $extraUtils/share/plymouth/plymouthd.defaults /etc/plymouth/plymouthd.defaults
       ln -s $extraUtils/share/plymouth/logo.png /etc/plymouth/logo.png
       ln -s $extraUtils/share/plymouth/themes /etc/plymouth/themes
       ln -s $extraUtils/lib/plymouth /etc/plymouth/plugins
+      ln -s $extraUtils/etc/fonts /etc/fonts
```

Comment:
This will break with #73795, since fontconfig now includes configuration from `${fontconfig}/etc/fonts/conf.d`, which is not in allowed references:

```
output '/nix/store/4gkpl38kjw3qskfz5gy7f74ahlps04c7-extra-utils' is not allowed to refer to the following paths:
  /nix/store/whh21mr17whyq1mm2h7lzj98ixi4w3ps-fontconfig-2.13.92
```